Persephone/Components/Queue/CurrentCoverArtView.swift

```diff
@@ -8,23 +8,39 @@
 
 import AppKit
 import ReSwift
+import Kingfisher
 
 class CurrentCoverArtView: NSImageView {
   required init?(coder: NSCoder) {
     super.init(coder: coder)
 
     App.store.subscribe(self) {
-      $0.select { $0.playerState.currentArtwork }
+      $0.select { $0.playerState.currentSong }
     }
   }
+
+  func setAlbumImage(_ album: Album) {
+    guard let imagePath = album.coverArtFilePath else { return }
+
+    let imageURL = URL(fileURLWithPath: imagePath)
+    let provider = LocalFileImageDataProvider(fileURL: imageURL)
+    self.kf.setImage(
```

Comment:
Can you eliminate the `self` here?

Reply:
As a follow-up, this is probably a style preference, but at least in the iOS Swift world, the impression I've gotten from other mobile developers at thoughtbot is that they try to remove `self` statements unless they're absolutely needed.

It's not listed in the [style guide](https://github.com/thoughtbot/guides/tree/master/style/swift), but maybe it should be if it's a thing?